pdfminer/pdffont.py

```diff
@@ -134,15 +133,11 @@ def do_keyword(self, pos, token):
            None, '-')
 
 # Note: DLIdent-* isn't found in PDF Reference but is been kept as
-# it is harmless and have possibility of been a type.
-# (induced from bug report/PR)
-IDENTITY_ENCODER = {'Identity-H': 'Identity-H',
-                    'Identity-V': 'Identity-V',
-                    'DLIdent-H': 'Identity-H',
-                    'DLIdent-V': 'Identity-V',
-                    'OneByteIdentityH': 'OneByteIdentityH',
-                    'OneByteIdentityV': 'OneByteIdentityV',
-                    }
+# it is harmless and might be used by some PDFs.
+IDENTITY_ENCODER = {
```

Comment:
I'd maybe add a comment to briefly say what this does - i.e. "Used to fix incorrect identities. Identities in this list will be mapped to the given values when getting the cmap".